### ISSUE TITLE
To reproduce the final model

### ISSUE THREAD

**Issue by jhkim0759** [Arthur151/ROMP#182]
In the paper, it says that fine tuned model shows the best performance at 120Epoch, but after running the v1_hrnet_3dpw_ft.yml file, it showed the best performance at 1 epoch. Did I do something wrong?

**Comment by Arthur151** [comment]
Sorry about that. There is some randomness in training, like random sample selection, random augmentation. If the model get trapped into some local minima or fitting to specific domain, you may adjust the sampling ratio of different datasets in config to avoid this.

**Comment by jhkim0759** [comment]
Thank you, 

And I have another question 
If I follow the config v1.yml and config v1_hrnet_3dpw_ft.yml, then can I reproduce the model as protocol 3*??

**Comment by Arthur151** [comment]
Yes, but you can also directly train from our pre-trained model with v1_hrnet_3dpw_ft.yml to have protocol 3, which is just train with 3DPW.

**Comment by jhkim0759** [comment]
Thank you, 

Is it protocol 3 or protocol 3*?? 
And, Are these two different in pretraining processes??

**Comment by Arthur151** [comment]
* if you choose to use more datasets during training than [EFT](https://arxiv.org/pdf/2004.03686.pdf)

**Comment by jhkim0759** [comment]
OK, I'll try it

I'm really thank you for your help.
